helpers/formatNumber.ts

```diff
@@ -1,13 +1,24 @@
 import { BigNumber } from "ethers";
-import { commify, formatEther } from "helpers";
+import { commify, formatEther, parseEther } from "helpers";
 
-export function formatNumberForDisplay(number: BigNumber | undefined, decimals = 2) {
+export function formatNumberForDisplay(
+  number: BigNumber | undefined,
+  options?: { decimals?: number; isFormatEther?: boolean }
+) {
   if (!number) return "0";
-  return truncateDecimals(commify(formatEther(number)), decimals);
+  const { decimals = 2, isFormatEther = true } = options || {};
+  const _number = isFormatEther ? formatEther(number) : number.toString();
+  return truncateDecimals(commify(_number), decimals);
 }
 
 export function truncateDecimals(number: string, decimals: number) {
   const [whole, decimal] = number.split(".");
   if (!decimal) return number;
+  if (decimals === 0) return whole.toString();
   return `${whole}.${decimal.slice(0, decimals)}`;
 }
+
+export function bigNumberFromFloatString(value: string) {
```

Comment:
id suggest having a decimal param that defaults to 18. though you are probably pretty safe assuming 18 everywhere in the voterdapp